test/examples/to-fix/llvm/Datatypes.ju

```diff
@@ -18,6 +18,37 @@ sig patternMatch : Bar -> int
 let patternMatch (P x y) = x + y
 let patternMatch (Q x y z) = x + y + z
 
+type FunctionTypeInConstructor :
```

Comment:
Nit : For consistency can we use lower cased type names?

Reply:
Sure, I've made the type names lowercase now.